### DISCUSSION TITLE
Indexes will not be recreated

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#11812] [Q&A] [answered]
Delete es index: DELETE /sw_*

After deleting the index, restart apm and webapp; the index will not be re-created;
I need to do this to recreate the index

**Comment by wu-sheng** [comment, marked as answer]
If this is running in k8s, this wouldn't be automatically.
Find the OAP init container job, and rerun that.

**Reply by mitht** [reply]
Deployed based on linux host

After deleting the index, restart using apache-skywalking-apm-bin/bin/startup.sh; The index is not re-created after restarting

vim skywalking-webapp.log

<img width="1681" alt="image" src="https://github.com/apache/skywalking/assets/29425860/fb745b8c-aedb-4de2-9d81-40d1d387b40d">

vim vim skywalking-oap-server.log

<img width="1903" alt="image" src="https://github.com/apache/skywalking/assets/29425860/f2203284-4960-418e-b8ef-5e47b1f6f81b">

kibana:

<img width="1605" alt="image" src="https://github.com/apache/skywalking/assets/29425860/6153de3d-24a5-4e92-a693-a597ef53de12">

**Reply by wu-sheng** [reply]
That means you keep the index templates. You need to delete them to trigger it.

**Reply by mitht** [reply]
Do I need to delete templates prefixed with sw?
<img width="1445" alt="image" src="https://github.com/apache/skywalking/assets/29425860/9fe77fc6-2aff-4cc0-a3af-88dc4b3f2cb3">

**Reply by wu-sheng** [reply]
Yes, you are.

**Reply by mitht** [reply]
Done, thank you!
